crates/biome_css_analyze/tests/specs/nursery/noUnknownUnit/invalid.css.snap.new

```diff
@@ -0,0 +1,703 @@
+---
```

Comment:
You committed this file by mistake. You have to run `cargo insta accept` to generate the snapshot

Reply:
Sorry. I committed the correct snapshot file.